src/lib/cards/MissionCompletionCard.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+	import type { IndividualCompletion } from '$lib/types';
+	import { getPersonColor, listify } from '$lib/util';
+
+	export let mission: IndividualCompletion;
+	export let username: string;
+	const methods = [
+		mission.defuser ? 'as a Defuser' : null,
+		mission.expert ? 'as an Expert' : null,
+		mission.efm ? 'via EFM' : null,
+		mission.solo ? 'solo' : null
+	].flatMap(method => method ?? []);
+	const title = `${username} solved this mission ${listify(methods)}.`;
+</script>
+
+<a class="mission" href="/mission/{encodeURIComponent(mission.name)}">
+	<div class="mission-name" class:green={mission.defuser && mission.expert && mission.efm}>
+		{mission.name}
+	</div>
+	<div {title}>
+		<div
+			class:hidden={!mission.defuser}
+			class:hspace={mission.nDefuser < 2}
+			style="background-color: {getPersonColor(2, 0, false)}">
+			<span class:hidden={mission.nDefuser < 2}>×{mission.nDefuser}</span>
+		</div>
+		<div
+			class:hidden={!mission.expert}
+			class:hspace={mission.nExpert < 2}
+			style="background-color: {getPersonColor(2, 1, false)}">
+			<span class:hidden={mission.nExpert < 2}>×{mission.nExpert}</span>
+		</div>
+		<div
+			class:hidden={!mission.efm}
+			class:hspace={mission.nEFM < 2}
+			style="background-color: {getPersonColor(1, 0, false)}">
+			<span class:hidden={mission.nEFM < 2}>×{mission.nEFM}</span>
+		</div>
+		<div
+			class:hidden={!mission.solo}
+			class:hspace={mission.nEFM < 2}
```

Comment:
I think this should be:
```suggestion
			class:hspace={mission.nSolo < 2}
```